### PR TITLE
more information

### PR DESCRIPTION
**Pull request by d1g**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +41,8 @@ it easier to detect mapping errors raising the **quality of the OSM data**.
 
 ## QA Tools
 
-* [OSM Inspector](http://tools.geofabrik.de/osmi/?view=areas)
+* [OSM Inspector](http://tools.geofabrik.de/osmi/?view=areas) - updare frequency?
```
Comment (pnorman) [inline README.md]: updare?

File: README.md [hunk 2]
```diff
@@ -2,12 +2,10 @@
...
-correctly and there are even more ways in which they can be mapped incorrectly.
-Of the over 220 million (multi)polygons in OSM more than 100,000 contain
-mapping errors of one kind or another and about 250,000 are tagged *old style*
-with tags on the outer ways instead of on the relation making multipolygon
-tagging and processing much more complicated and much more expensive than it
-needs to be.
+correctly and there are even more ways in which they can be mapped incorrectly, 
+using *old style* - with tags on the outer ways instead of on the relation 
+making multipolygon tagging and processing much more complicated and much more 
+expensive than it needs to be.
```
Comment (pnorman) [inline README.md]: This is changing the meaning of the text. I'd rather add information about the number of _new style_ with tags on the relation.

Comment (d1g) [inline README.md]: @pnorman, my point was that _exact numbers_ will be misleading in txt file, we should use (near) real-time stats if possible; I have no concerns about text, but outdated numbers are very misleading.

File: README.md (hunk 1 above)
Comment (d1g) [inline README.md]: fixed typo, but I still can't find info about update frequency

File: README.md (hunk 2 above)
Comment (joto) [inline README.md]: @d1g Those numbers are here to give an idea of the scale of the problem. They are, of course, not exact. But they are useful the way the are to show that this isn't a small problem. Daily updated numbers are available from the stats page.

Comment (d1g) [inline README.md]: @joto reverted textual changes. Can we please have [more links and sections](https://github.com/osmlab/fixing-polygons-in-osm/pull/13/files) in README? 

BTW you seem to be a right person to publish an info about update frequency :)

Comment (d1g) [inline README.md]: @pnorman, @joto - I have to explain what I meant by "very misleading"

"[Of the over 220 million (multi)polygons in OSM](https://github.com/osmlab/fixing-polygons-in-osm#fixing-polygons-in-osm)" - but if you check a database you [will get over 4M relations and only 2328618 of them are type=multipolygon](https://taginfo.openstreetmap.org/relations).

So either: 1. you clarify how do you calculate "relation" or "(multi)polygons" or rather use real metric "number of ways without area=no tags + valid MP" or 2. remove unannotated (improperly described) numbers/stats

Thats why I'm suggesting to include links to database(s) and [stats](http://area.jochentopf.com/stats/).

**Pull request merged**